--- dsktool-1.0.2/SerialForm.cpp.orig	2007-04-23 21:53:06.000000000 +0200
+++ dsktool-1.0.2/SerialForm.cpp	2007-05-02 12:18:10.566691398 +0200
@@ -175,7 +175,7 @@
 		m_string += wxT(",");
 		m_string += m_typebox->GetValue();
 	}
-#if (WXMAJOR_VERSION > 2) || (WXMAJOR_VERSION == 2 && WXMINOR_VERSION > 6) 
+#if (wxMAJOR_VERSION > 2) || (wxMAJOR_VERSION == 2 && wxMINOR_VERSION > 6) 
 	EndDialog(wxID_OK); 
 #else
 	wxDialog::OnOK(ev);
